src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
-// write code here
+function convertToNumber(value) {
+  return Number(value);
+}
+
+// Функція для сортування списку за зарплатою в порядку спадання
+// eslint-disable-next-line no-shadow
+function sortList(list) {
```

Comment:
The `sortList` function is correctly defined to sort the list of employees by salary. However, ensure that the `list` parameter is a valid DOM element with children having the necessary data attributes (`data-salary`).